webui/react/src/utils/sort.ts

```diff
@@ -7,18 +7,19 @@ import { NullOrUndefined, Primitive } from '../shared/types';
 const runStateSortValues: Record<RunState, number> = {
   [RunState.Active]: 0,
   [RunState.Paused]: 1,
-  [RunState.StoppingError]: 2,
-  [RunState.Errored]: 3,
-  [RunState.StoppingCompleted]: 4,
-  [RunState.Completed]: 5,
-  [RunState.StoppingCanceled]: 6,
-  [RunState.Canceled]: 7,
-  [RunState.Deleted]: 7,
-  [RunState.Deleting]: 7,
-  [RunState.DeleteFailed]: 7,
-  [RunState.Insufficient]: 8,
-  [RunState.Unspecified]: 9,
-  [RunState.Pending]: 10,
+  [RunState.Pending]: 2,
```

Comment:
assuming typescript can still infer the types:
could we generate this off of an ordered array? that'd make the diffs more readable.

Reply:
I've just added a commit to have an array of RunStates, and store them and an index in a JavaScript Map object. Same for CommandStates.